frontend/packages/vkt/src/components/clerkEnrollment/listing/UpdateEnrollmentStatusButton.tsx

```diff
@@ -0,0 +1,80 @@
+import { ChevronRight } from '@mui/icons-material';
+import { CustomButton, LoadingProgressIndicator } from 'shared/components';
+import { APIResponseStatus, Color, Variant } from 'shared/enums';
+
+import { useClerkTranslation } from 'configs/i18n';
+import { useAppDispatch, useAppSelector } from 'configs/redux';
+import { EnrollmentStatus } from 'enums/app';
+import { ClerkEnrollment } from 'interfaces/clerkExamEvent';
+import { updateClerkEnrollmentStatus } from 'redux/reducers/clerkExamEventOverview';
+import { clerkExamEventOverviewSelector } from 'redux/selectors/clerkExamEventOverview';
+
+export const UpdateEnrollmentStatusButton = ({
+  enrollment,
+}: {
+  enrollment: ClerkEnrollment;
+}) => {
+  const { t } = useClerkTranslation({
+    keyPrefix:
+      'vkt.component.clerkEnrollmentListing.row.updateEnrollmentStatus',
```

Comment:
updateEnrollmentStatus -> changeStatus (tai sitten jättää vaan näin)

Reply:
`changeEnrollmentStatus` tuntu osuvan tuohon parhaiten